### ISSUE TITLE
Error 500 on older iOS Safaris in the default Sapper template app

### ISSUE THREAD

**Issue by arggh** [sveltejs/sapper#581]
I've been testing a Sapper app on older browsers and found a peculiar case. An app that runs just fine on IE 11 will not run properly on iOS 9.x. I'm having a hard time catching the actual error.

Can be reproduced easily just by starting with the default Sapper template and then adding the missing polyfills (fetch?), which is exactly what I've done in this [reproduction](https://github.com/arggh/sapper-ios-issue).

Without the polyfills, the JavaScript execution crashes on startup due to missing fetch and from there on the site works as a bunch of static html files. It's browsable, but only as a static html page. 

**However**, after adding the polyfills, the site is basically **unusable**. Sapper's error page is shown with message `500 - undefined` immediately after the javascript kicks in, which means the user cannot even browse the static content or navigate pages.

This is a bummer, as it's not _that_ un-likely for decent normal people to be browsing the web with completely functional iPads that are _stuck_  on iOS 9.x or 10.x.

To recap, if a Sapper site works in IE 11 (released 2013), I would expect it to work in iOS 9 (2015). If not, there has to be something fishy going on, right?

This issue seems related: https://github.com/sveltejs/sapper/issues/464

### How to run the repro

1. Clone the repository
2. `npm install`
3. `npm run build`
4. `npm run start`
5. Open the running site on an iPad with iOS 9.x
- You can use the XCode Simulator on Mac OS. It's a little tricky to install the older iOS versions, as it the correct location is in Hardware > Devices > Manage Devices > Simulators > + > OS Version [Download more simulator runtimes]
6. After opening the site with iOS 9/10, You will be greeted with `Error 500` displayed after a brief inital flash of the static rendered content

**Comment by arggh** [comment]
Just a quick update, I tested also iOS 10.0 - 10.2 versions which also need the fetch polyfill, they all suffer from this same issue.

I also tested a few different polyfills, all the same.

**Comment by artemjackson** [comment]
@arggh 
I had a similar issue. 

In my case it was caused by `svelte/shared.js` which is written in es6 (which is not supported by Safari 9).

To force svelte internal stuff to be transpiled to es5 via babel I had to add a [project-wide](https://babeljs.io/docs/en/config-files#project-wide-configuration) `babel.config.js`

Get a look, may be it will help you

**Comment by arggh** [comment]
Apparently this had something to do with the fetch implementation from polyfill.io. Seems working now ¯\_(ツ)_/¯
